rest/data/repotracker.go

```diff
@@ -93,9 +108,11 @@ func (c *MockRepoTrackerConnector) TriggerRepotracker(_ amboy.Queue, _ string, e
 	if err != nil {
 		return err
 	}
+	if len(branch) == 0 {
```

Comment:
what does this case cover?

Reply:
if it's not a commit push (such as a tag push), branch will be empty

Reply:
These aren't errors, so i didn't return an error.